pages/login.js

```diff
@@ -10,6 +10,20 @@ const Title = styled.div`
   font-weight: bold;
 `;
 
+const { detect } = require("detect-browser");
+const browser = detect();
+
+// handle the case where we don't detect the browser
+switch (browser && browser.name) {
+  case "ie":
+    alert(
+      "Votre navigateur n'est pas compatible avec l'application e-mjpm. Nous vous recommandons d'utiliser Firefox ou Chromium ou Chrome."
+    );
+    break;
+  default:
+    console.log("not supported");
+}
+
```

Comment:
create a dedicated page with a nicer message and a link to download firefox for IE users  and redirect them here